ui/temboardui/templates/settings/instance.html

```diff
@@ -53,7 +54,15 @@
             </div>
           </td>
           <td>{{instance.pg_data}}</td>
-          <td>{% for group in instance.groups %} <span class="badge border text-muted">{{group.group_name}}</span>{% end %}</td>
+          <td>
+            {% for group in instance.groups %} <span class="badge border text-muted">{{group.group_name}}</span>{% end %}
+            {% if len(instance.groups) > 1 %}
+            <button
+              id="buttonMigrationGroups" class="btn btn-warning btn-sm ml-1" data-toggle="modal" v-on:click.prevent="$refs.migration_groups_wizard.open('instance_groups')">
+              <i class="fa fa-warning"></i>
```

Comment:
- Supprimer le bouton.
- Mettre directement `v-on:click=` sur `<i>`.
- Ajouter `text-warning` à `<i class=>`.
- Ajouter `style="cursor: pointer;` à `<i>`.
- Ajouter `title="Instance in multiple groups. Click for details."`